docs/content.zh/docs/connectors/table/filesystem.md

```diff
@@ -88,95 +86,101 @@ path
         ├── part-0.parquet
 ```
 
-The file system table supports both partition inserting and overwrite inserting. See [INSERT Statement]({{< ref "docs/dev/table/sql/insert" >}}). When you insert overwrite to a partitioned table, only the corresponding partition will be overwritten, not the entire table.
+文件系统表支持分区新增插入和分区覆盖插入。请参考 [INSERT Statement]({{< ref "docs/dev/table/sql/insert" >}}) 。当对分区表进行分区覆盖插入时，只有相应的分区会被覆盖，而不是整个表。
+
+<a name="file-formats"></a>
 
-## File Formats
+## 文件格式
```

Comment:
IMO,  keep the original text and keep ‘Format(s)’ consistent in the whole article.
cc @wuchong @Thesharing